OutOfSchool/OutOfSchool.WebApi.Tests/Controllers/AdminControllerTests.cs

```diff
@@ -0,0 +1,956 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Security.Claims;
+using System.Threading.Tasks;
+using AutoMapper;
+using FluentAssertions;
+using Microsoft.AspNetCore.Authentication;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Filters;
+using Microsoft.Extensions.Localization;
+using Microsoft.Extensions.Logging;
+using Moq;
+using NUnit.Framework;
+using OutOfSchool.Common;
+using OutOfSchool.Common.Enums;
+using OutOfSchool.Services.Enums;
+using OutOfSchool.Services.Models;
+using OutOfSchool.Tests.Common;
+using OutOfSchool.Tests.Common.TestDataGenerators;
+using OutOfSchool.WebApi.Common;
+using OutOfSchool.WebApi.Controllers.V1;
+using OutOfSchool.WebApi.Models;
+using OutOfSchool.WebApi.Models.Application;
+using OutOfSchool.WebApi.Models.Providers;
+using OutOfSchool.WebApi.Models.SocialGroup;
+using OutOfSchool.WebApi.Models.Workshops;
+using OutOfSchool.WebApi.Services;
+using OutOfSchool.WebApi.Util;
+using OutOfSchool.WebApi.Util.Mapping;
+
+namespace OutOfSchool.WebApi.Tests.Controllers;
+
+[TestFixture]
+public class AdminControllerTests
+{
+    private AdminController controller;
+    private Mock<ICurrentUserService> currentUserService;
+    private Mock<ISensitiveMinistryAdminService> sensitiveMinistryAdminService;
+    private Mock<ISensitiveDirectionService> sensitiveDirectionService;
+    private Mock<ISensitiveProviderService> sensitiveProviderService;
+    private Mock<ISensitiveApplicationService> sensitiveApplicationService;
+    private Mock<ILogger<AdminController>> logger;
+    private Mock<IStringLocalizer<SharedResource>> localizer;
+
+    private string userId;
+    private Guid providerId;
+    private Mock<HttpContext> httpContext;
+
+    private List<ApplicationDto> applications;
+    private IEnumerable<ChildDto> children;
```

Comment:
Field `providers` isn't used as well

Reply:
Closing this one (mainly because I'm tired and see no good progress), though neither `provider`, nor `userRole` fields (both redundant) are not removed. 
Kyrylo you are very inattentive (